src/create/index.js

```diff
@@ -20,6 +20,14 @@ const templates = glob(path.join(__dirname, "*.template")).map(f => path.parse(f
 module.exports = {
 	command: "create <module>",
 	describe: `Create a Moleculer module (${templates.join(",")})`,
+	builder(yargs) {
+		yargs.options({
+			"typescript": {
+				describe: "create service for typescript",
```

Comment:
Please use capital letter. It should be `Create` and not `create`. Also, the correct way is `TypeScript`